aspnetcore/blazor/host-and-deploy/configure-linker.md

```diff
@@ -2,12 +2,12 @@
 title: Configure the Linker for ASP.NET Core Blazor
 author: guardrex
 description: Learn how to control the Intermediate Language (IL) Linker when building a Blazor app.
-monikerRange: '>= aspnetcore-3.1'
+monikerRange: '>= aspnetcore-3.1 && <=aspnetcore-5.0'
```

Comment:
Most of the content for the 3.2 linker does not apply to 5.0. I wasn't sure if I could have two pages with the same uid

Reply:
I have a feeling this is going to break the doc build system, but it's never been tried before. I'll look at the build report and see how it went.

Reply:
Yeah .... thought so ....

# :boom: :cry:

> moniker-overlapping ... Two  or more documents with the same uid  `blazor/host-and-deploy/configure-trimmer`('blazor/host-and-deploy/configure-linker.md',  'blazor/host-and-deploy/configure-trimmer.md') have defined overlapping  moniker: 'aspnetcore-5.0'.

I'll create a new PR that does this on Monday, then I'll ping u to review it to make sure that it's all good.